docs/source/reference/config-user-env.md

```diff
@@ -43,6 +43,8 @@ sudo python3 -m pip install numpy
 to install the numpy package in the default system Python 3 environment
 (typically `/usr/local`).
 
+TODO: Get a link from the conda team for a description of what "appropriate permissions for users" is
+
 You may also use conda to install packages. If you do, you should make sure
 that the conda environment has appropriate permissions for users to be able to
 run Python code in the env.
```

Comment:
The appropriate permissions are described in the previous paragraph:

> ... must be *readable and executable* by your users. If you want users to be able to install additional
> packages, it must also be *writable* by your users.

The appropriate permissions are not sensitive to which installation tools are used.